<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizza Utente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xz.qrc">
    <normaloff>:/images/images/AnconAmbiente_Logo.png</normaloff>:/images/images/AnconAmbiente_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PERMESSI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>231</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Clienti:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>55</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>55</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clienti_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="servizi_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="turni_check">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>199</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>171</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Operatori:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Utenti</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="utenti_check">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>229</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="annulla_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>INDIETRO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Turni</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mezzi_check">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>169</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>201</y>
      <width>80</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Servizi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mezzi</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="operatori_check">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="salva_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>MODIFICA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>65</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>105</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>25</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xz.qrc"/>
 </resources>
 <connections/>
</ui>
